### PR TITLE
Feature/plot fiber types in layers

### PR DESCRIPTION
**Pull request by ChrisMolanus**
Make sure that when we plot the cabinet to DC cables are visible without breaks

### CODE REVIEW

File: fibers.py [hunk 1]
```diff
@@ -675,18 +675,20 @@ def plot_fiber_network(fiber_graph, fiber_dc_graph, building_gdf, cabinet_look_u
...
-    ax.scatter(cabinet_df.x, cabinet_df.x, s=30, color="m")
-    ax.scatter(ds_df.x, ds_df.y, s=70, color="yellow")
 
     fig, ax = ox.plot_graph(fiber_dc_graph, bgcolor=None, edge_color="lime",
                             node_size=0, edge_linewidth=2.5, edge_alpha=0.8,
                             show=False, close=False, ax=ax)
 
+    fig, ax = ox.plot_footprints(building_gdf, ax=ax, color="burlywood", alpha=0.6, show=False, close=False)
+
+    ax.scatter(cabinet_df.x, cabinet_df.x, s=70, color="red")
```
Comment (tivoo) [inline fibers.py]: second cabinet_df.x should be cabinet_df.y

**Review by tivoo — CHANGES_REQUESTED** [review]
If we change scatterplot to correct y coords it should work this way with two separate fiber graphs. Let's stick with this merge, because my version still does not render properly.

**Comment by tivoo** [comment]
Also, in fiber_network.equipment EquipmentType.StreetCabinet is actually empty, causing the costs to not show any of the street cabinets. That still needs to be fixed as well

**Review by HSCunha — APPROVED** [review]

File: fibers.py (hunk 1 above)
Comment (ChrisMolanus) [reply fibers.py]: Fixed

**Review by tivoo — APPROVED** [review]
Works now, approved.

**Pull request merged**